improver_tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -184,46 +182,30 @@ def test_radii_varying_with_lead_time_mismatch(self):
         lead_times = [2, 3]
         msg = "There is a mismatch in the number of radii"
         with self.assertRaisesRegex(ValueError, msg):
-            neighbourhood_method = Neighbourhood("circular")
-            NBHood(neighbourhood_method, radii, lead_times=lead_times)
+            NBHood(radii, lead_times=lead_times)
 
 
 class Test__find_radii(IrisTest):
 
     """Test the internal _find_radii function is working correctly."""
 
-    def test_basic_float_cube_lead_times_is_none(self):
```

Comment:
Changed it so if no leadtimes are provided then the radius is stored at the init step, so this test isn't needed now.